### DISCUSSION TITLE
[Question] rke2 vs k3s

### DISCUSSION THREAD

**Discussion by ceyhunn** [vitobotta/hetzner-k3s#250] [Q&A] [answered]
Hi @vitobotta ,

What is the best kubernetes distribution for hetzner? I research on internet , k3s is the suitable for edge computing and Iot. Can I use k3s with 50 -100 workers in hetzner cloud? I want to install rancher, that is why which one to choose rke2 or k3s?

**Comment by vitobotta** [comment, marked as answer]
Hi, please use Discussions for things other than issues with the tool.

k3s is very lightweight which makes it ideal for devices with limited resources, but it's a fully conformant Kubernetes distribution and can be used for large clusters with big nodes etc if you want. It's entirely up to you.

RKE2 is very similar to k3s. AFAIK the biggest difference is some advanced security features that might make it more appealing for things like governments etc. But k3s is rock solid and easy to operate.

**Comment by ceyhunn** [comment]
Wow, thank you for great explanation.
